lib/Bio/EnsEMBL/DataCheck/Checks/CheckGenomicAlignGenomeDBs.pm

```diff
@@ -0,0 +1,91 @@
+=head1 LICENSE
+
+Copyright [2018-2019] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the 'License');
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an 'AS IS' BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package Bio::EnsEMBL::DataCheck::Checks::CheckGenomicAlignGenomeDBs;
+
+use warnings;
+use strict;
+
+use Moose;
+use Test::More;
+use Bio::EnsEMBL::DataCheck::Test::DataCheck;
+use Bio::EnsEMBL::DataCheck::Utils qw/ array_diff /;
+
+extends 'Bio::EnsEMBL::DataCheck::DbCheck';
+
+use constant {
+  NAME           => 'CheckGenomicAlignGenomeDBs',
+  DESCRIPTION    => 'Check all genome_dbs for each method_link_species_set is present in genomic_aligns',
+  GROUPS         => ['compara', 'compara_multiple_alignments', 'compara_pairwise_alignments'],
+  DATACHECK_TYPE => 'critical',
+  DB_TYPES       => ['compara'],
+  TABLES         => ['dnafrag', 'genome_db', 'genomic_align', 'genomic_align_block', 'method_link_species_set', 'species_set']
+};
+
+sub tests {
+  my ($self) = @_;
+  my $dba = $self->dba;
+  my $helper = $dba->dbc->sql_helper;
+  my $mlss_adap = $dba->get_MethodLinkSpeciesSetAdaptor;
+  my $gdb_adap = $dba->get_GenomeDBAdaptor;
+  my @mlss_types = qw ( PECAN EPO EPO_LOW_COVERAGE CACTUS_HAL LASTZ_NET LASTZ_PATCH);
```

Comment:
```suggestion
  my @mlss_types = qw ( PECAN EPO EPO_LOW_COVERAGE LASTZ_NET LASTZ_PATCH);
```
Don't include HAL here - those blocks are generated on the fly and do not go in the database